db/schema.rb

```diff
@@ -0,0 +1,58 @@
+# This file is auto-generated from the current state of the database. Instead
```

Comment:
I'm not sure if this file should be in the repository, could you please confirm @duduribeiro? I think this is a reminiscent of the workaround to run using Docker.

Reply:
you are right, this should be removed. @rafaelsevla can you remove this file and recommit again?